### PR TITLE
MAN-28-Create-GitHub-Actions-CI-pipelines-for-the-Frontend-SPA

### PR DESCRIPTION
**Pull request by cant0r**

### CODE REVIEW

File: .github/workflows/build-all-and-validate-nightly.yml [hunk 1]
```diff
@@ -33,8 +33,11 @@ jobs:
             .nuke/temp
             ~/.nuget/packages
           key: ${{ runner.os }}-${{ hashFiles('**/global.json', '**/*.csproj') }}
-      - name: Run './build.cmd CompileAll ValidateCLI'
-        run: ./build.cmd CompileAll ValidateCLI
+      - name: Run './build.cmd CompileAll ValidateCLI PublishAll'
+        run: ./build.cmd CompileAll ValidateCLI PublishAll
+        env:
+          DOCKER_USERNAME: ${{ secrets.DOCKER_USERNAME }}
```
Comment (TheUsernameIsNotTaken) [inline .github/workflows/build-all-and-validate-nightly.yml:39]: Secrets Good.

File: docker-compose.yml [hunk 2]
```diff
@@ -0,0 +1,11 @@
+version: "3.9"
+services:
+  backend:
+    image: "youtoddler/backend:latest"
+    command: java -jar youtoddler-spring-0.7.2.jar
```
Comment (TheUsernameIsNotTaken) [inline docker-compose.yml:5]: Yeah, this is what I in the last one. At least a comment will need to be added in my soruce code, but let's discuss it next meeting,

**Review by TheUsernameIsNotTaken — CHANGES_REQUESTED** [review]
Version fixed in value is needed?

File: docker-compose.yml (hunk 2 above)
Comment (TheUsernameIsNotTaken) [reply docker-compose.yml:5]: Ok, is it ok if I try the changes mentioned #27 here first?

**Review by TheUsernameIsNotTaken — APPROVED** [review]

**Review by szemand — APPROVED** [review]

**Pull request merged**